eng/Versions.props

```diff
@@ -38,7 +37,7 @@
   <PropertyGroup>
     <!-- arcade -->
     <MicrosoftDotNetBuildTasksPackagingPackageVersion>1.0.0-beta.19308.1</MicrosoftDotNetBuildTasksPackagingPackageVersion>
-    <MicrosoftDotNetBuildTasksFeedPackageVersion>2.2.0-beta.19308.1</MicrosoftDotNetBuildTasksFeedPackageVersion>
+    <MicrosoftDotNetBuildTasksFeedPackageVersion>2.2.0-beta.19410.2</MicrosoftDotNetBuildTasksFeedPackageVersion>
```

Comment:
I filed https://github.com/dotnet/core-setup/issues/7693 to get these up to date more generally and get them auto-updating.